README.md

```diff
@@ -297,6 +297,27 @@ of mangled property names.
 Using the name cache is not necessary if you compress all your files in a
 single call to UglifyJS.
 
+When using `minify()`, you may specify `--name-cache` in the following way:
```

Comment:
Can you move this entire example into the API section below the in-memory `nameCache` example?
```
When using `minify()`, you may persist the name cache to the file system in the following way:
```